### DISCUSSION TITLE
Github student developer pack processing information error

### DISCUSSION THREAD

**Discussion by bakjinsoo** [community/community#84813] [GitHub Education] [answered]
### Hi everyone,

Hello, I'm trying to get a student authentication, but after 3 errors appear, submit your student ID and press the processing information button, it won't move on from this status.

**Comment by MarcoMihalca** [comment]
How did you fix it?

**Comment by SamoZ256** [comment]
There is a bug and the image you provide needs to have less than 150 kb.

**Comment by mishmanners** [comment, marked as answer]
The same is resolved here: https://github.com/orgs/community/discussions/88401#discussioncomment-8142514 and here: https://github.com/orgs/community/discussions/88446#discussioncomment-8148738

Please see these. The TL;DR:
- Refresh the page or clear your browser's cache and cookies
- Check for browser extensions that could be interfering with the JavaScript code and disable them
- Try a different browser
- Ensure your file less than 150kB
